blivet/formats/fs.py

```diff
@@ -274,32 +274,25 @@ def update_size_info(self):
         self._min_instance_size = Size(0)
         self._resizable = self.__class__._resizable
 
-        # We can't allow resize if the filesystem has errors.
+        # try to gather current size info
```

Comment:
Assuming this goes in, method comments in header should be updated to match code changes.